livre/eq/Node.cpp

```diff
@@ -129,6 +119,17 @@ class Node
         _textureDataCachePtr.reset();
     }
 
+    void updateAndSendFrameRange()
```

Comment:
Agreed, and I'm saying updateFrameRange() is going on.

Was just a comment, not a request. I prefer concise names, Java typically has longer names which almost implement themselves. YMMV. ;)